src/Domain/TagsTimeDependent.hpp

```diff
@@ -0,0 +1,153 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <boost/optional.hpp>
+#include <cstddef>
+#include <memory>
+#include <string>
+#include <tuple>
+#include <unordered_map>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"  // For Tags::Normalized
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/FaceNormal.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "Domain/FunctionsOfTime/Tags.hpp"
+#include "Domain/Tags.hpp"
+#include "Time/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace domain {
+/// \ingroup ComputationalDomainGroup
+/// \brief %Tags for the domain.
+namespace Tags {
+/// The Inertial coordinates, Inertial grid velocity, and the Jacobian and
+/// inverse Jacobian from the Grid to the Inertial frames.
```

Comment:
The order of quantities doesn't correspond to the list of types. Also the documentation of `CoordinateMapBase::coords_frame_velocity_jacobians` doesn't list the quantities in the right order. Could you make these consistent please

Also, below you have named a tag `FrameVelocity`, so this should probably be named `FrameVelocityAndJacobians ` for consistency

Reply:
I've decided to rename `FrameVelocity` to `MeshVelocity` and made the consistency change you suggested